src/less/donut-chart/stories/donut-chart.stories.js

```diff
@@ -0,0 +1,55 @@
+export default { title: 'Donut-chart' };
+
+export const basic = () => `
```

Comment:
The colors look off to me. 
<img width="527" alt="image" src="https://user-images.githubusercontent.com/1675667/199601827-9feb3f0e-e2aa-4960-8589-dee1d90da925.png">